src/main/kotlin/com/hypto/iam/server/service/CredentialService.kt

```diff
@@ -56,8 +56,8 @@ class CredentialServiceImpl : KoinComponent, CredentialService {
         organizationId: String,
         userId: String
     ): ListCredentialResponse {
-        val userHrn = ResourceHrn(organizationId, "", IamResources.USER, userId)
-        return ListCredentialResponse(repo.getAllByUserHrn(userHrn).map { Credential.from(it) })
+        val userHrn = ResourceHrn(usersService.getUser(organizationId, userId).hrn)
```

Comment:
Does `usersService.getUser` call AWS or just checks out DB?

Reply:
It only calls AWS.